src/main/java/pl/edu/agh/ki/io/api/PostApiController.java

```diff
@@ -33,7 +33,7 @@
 
     @PostMapping("/post")
     public ResponseEntity<PostResponse> createPost(@RequestBody CreatePostRequest postRequest, @AuthenticationPrincipal User user) {
-        Post post = new Post(user, postRequest.getContent());
+        Post post = new Post(user, postRequest.getContent(), postRequest.getLatitude(), postRequest.getLongitude());
```

Comment:
Jeśli będzie nullem to przejdzie to? nie wiem czy każdy post musi mieć współrzędne

Reply:
Nawet jak tworzy się post przez `new Post(user, postRequest.getContent())` to te współrzędne się ustawiają na 0.0. Jak wyślę z frontu null to też się na 0.0 ustawiają.